src/bp-media/bp-media-filters.php

```diff
@@ -296,6 +296,20 @@ function bp_media_update_activity_media_meta( $content, $user_id, $activity_id )
 		return false;
 	}
 
+	// Add in description in attachment when only one media uploaded.
+	if ( is_array( $_POST['media'] ) && 1 === count( $_POST['media'] ) ) {
+		if ( '' === trim( $content ) && ! empty( $activity_id ) ) {
+			$get_content = new BP_Activity_Activity( $activity_id );
+			$content = $get_content->content;
+		}
```

Comment:
@chetansatasiya we don't need to get content from the activity here. we already get the content in function arguments `( $content, $user_id, $activity_id )` if in there `$content` is empty we dont go into the condition and add attachment content.

Reply:
@mehulkaklotar DONE 👍